src/core/server/elasticsearch/elasticsearch_service.test.ts

```diff
@@ -125,7 +126,7 @@ describe('#setup', () => {
       const config = MockClusterClient.mock.calls[0][0];
       expect(config).toMatchInlineSnapshot(`
 Object {
-  "healthCheckDelay": 2000,
+  "healthCheckDelay": "PT2S",
```

Comment:
What is the significance of "PT2S"?

Reply:
The config validation was changed from number to duration. Then we parse these durations with momentjs so a user can specify "10ms" or "1h" etc. Once parsed, moment serialises values using https://en.wikipedia.org/wiki/ISO_8601#Durations